quicklogic/README.md

```diff
@@ -0,0 +1,199 @@
+# SymbiFlow for Quicklogic FPGAs
+
+## Quickstart guide
+
+### 1. Install Yosys with QuickLogic support
```

Comment:
This should probably need to rework this. 

The Yosys should be installed from conda like we do with the other tools.

Reply:
I agree. But currently there is a Yosys fork with QuickLogic support that would conflict with the version used in SymbiFlow. How do you propose to solve that until the QuickLogic support is in upstream Yosys?

Maybe create a separate conda env to be used with QuickLogic devices?

Reply:
I've updated the README.md so that it reflects the current QuickLogic support state to be merged.